packages/atlas/src/views/viewer/ViewerLayout.tsx

```diff
@@ -80,9 +80,15 @@ export const ViewerLayout: FC = () => {
                   <Route key={route.path} {...route} />
                 ))}
                 <Route
-                  path={relativeRoutes.viewer.editMembership()}
+                  path={relativeRoutes.viewer.memberSettings()}
                   element={
-                    <PrivateRoute isAuth={isLoggedIn} element={<EditMembershipView />} redirectTo={ENTRY_POINT_ROUTE} />
+                    // todo uncomment and use private route once authentication is handled
+                    // <PrivateRoute
+                    //   isAuth={isLoggedIn}
+                    //   element={<MembershipSettingsView />}
+                    //   redirectTo={ENTRY_POINT_ROUTE}
+                    // />
```

Comment:
Currently is impossible to sign in. For convenience, I commented PrivateRoute. If you want to test the view go to `/member/settings?tab=Public+profile`